### PR TITLE
Fix Debian's autopkgtest

### PR DESCRIPTION
**Pull request by rbalint**

### CODE REVIEW

File: debian/tests/upgrade-all-security [hunk 1]
```diff
@@ -53,6 +54,12 @@ chroot_exec "$chroot_dir" bash -c 'apt install -y /tmp/unattended-upgrades_*deb'
 
 
 enable_pocket "$chroot_dir" security
+chroot_exec "$chroot_dir" apt-get update
+if ! chroot_exec "$chroot_dir" apt list --upgradable 2> /dev/null | grep -q .; then
```
Comment (mvo5) [inline debian/tests/upgrade-all-security]: Silly question - the `grep -q .` is to check if there is any output in apt list --upgradable? If so would it maybe make sense to use something like "wc" to make this more explicit? Also when I run this here with an empty upgrade set I get: 
```
LANGUAGE=en apt list --upgradable 2>/dev/null |grep  .  && echo grep-successful
Listing...
grep-successful
```
is that expected?

Comment (julian-klode) [reply debian/tests/upgrade-all-security]: This does not work, apt list --upgradable include progress output in stdout.

Comment (julian-klode) [reply debian/tests/upgrade-all-security]: Can you use apt dist-upgrade -s | grep -q ^Inst instead?

Comment (rbalint) [reply debian/tests/upgrade-all-security]: @mvo5 @julian-klode Thanks, fixing it.

Comment (rbalint) [reply debian/tests/upgrade-all-security]: Fixed.

**Pull request merged**